README.md

```diff
@@ -25,3 +25,19 @@ the filename of `firstname-lastname.md` will allow hugo to automatically pull
 the testimonial student's name out. Once you run the command there will be a
 new file in `site/content/testimonials/` with some placeholder content. Edit
 that file to add the content for that student.
+
+## homepage promo images
+
+These are checked into the repo, stored as Markdown files managed by Hugo. To
+create a new one you can use `hugo new` like this:
+
+```
+hugo -s site new promos/filename.md
+```
+
+You'll then also need to check the image file into
+`site/static/images/promo-carousel/` and add it to the front matter in the file
+you just created, alongside the URL for the promo.
+
+You'll also need to set a background color to show behind the image on wide
```

Comment:
No need for background color anymore ?

Reply:
nope

Reply:
forgot to update the README, should be accurate now